### DISCUSSION TITLE
Site perigoso

### DISCUSSION THREAD

**Discussion by yagosantosog** [community/community#106955] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Eu coloquei um projeto no ar e quando acesso o link o navegador diz que o site é perigoso, mas não tem nada de perigoso no repositório.

**Comment by queenofcorgis** [comment]
Hi @yagosantosog , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
